### PR TITLE
Features

### PR DESCRIPTION
**Pull request by alexawesomecode**
Hello

Everything is working:

1. Use only JS to create elements and css
2. Different js files for tab
3. tab animation done in js
4. cool layout

### CODE REVIEW

File: src/onload.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 function onLoad() {
+  // content styling
```
Comment (bellom) [inline src/onload.js]: Kindly move this to CSS files
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/
Under General.

File: src/setStyle.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+
+const setStyle = () => {
+  const stylings = document.createElement('style');
+  document.querySelector('head').appendChild(stylings);
+
+  document.querySelector('style').innerHTML = `
+
+ .central p {
```
Comment (bellom) [inline src/setStyle.js]: Let's have our CSS styling in CSS file, we shouldn't write JS and CSS together.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/
 Check Under General.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @alexawesomecode  :clap: 

### Awesome job so far!! :+1:  

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
1. **May I know the reason why am seeing commits from two different GitHub users, seem @aalexu84 ain't one of the MIcroverse students. (I need to get back to you on this).**

2. **Kindly check my comments below.**

3. **You need to fix the linter's errors. and solve the merge conflicts**
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

Note: Do not make changes to the given config file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/onload.js (hunk 1 above)
Comment (alexawesomecode) [reply src/onload.js]: Hello @bellom

Thanks for your insightful review, it was very clear, I will work on the changes.

About the issue why appears another user as doing a commit (alexu84): I did this commit after I re-installed my Debian Linux Machine. I did it without configuring the git user.email or git.user, so by my mistake used the default user and domain from my machine "alex@debian" to the commit.

I have already had this problem before, on a previous project, after I re-installed my machine and forgot to set the credentials.

I thought It was a bug that you could just change your email, commit and it appears as the other person did the commit. I played with this doing commits from various people in another repo. I reported this to github but its a feature built it on github, indeed!

I have tried and tried to amend the commit name and email but I still coulnt figure it out

Sorry for the confusion

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @alexawesomecode 

nice interface and shaky animation
Screen Shot 2020-03-07 at 12 31 22 AM

I guess there's one debugging to do, the first menu doesn't work, when i click on `"DELICIOUS DISHES"`, it doesn't load and i get this console error, you can see from screenshot

### Let's get to debugging......

**Review by bellom — APPROVED** [review]
## Hi @alexawesomecode  :clap: 

**Great work making all the necessary corrections from the previous review.**

**Note:**Make sure to fix conflicts before submitting for review next time.

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**